spec/services/aspace_client_spec.rb

```diff
@@ -66,6 +66,24 @@
     end
   end
 
+  describe '#all_published_resource_uris_by' do
+    let(:published_resource_uris) do
+      [{ 'ead_id' => 'abc123', 'uri' => '/repositories/1/resources/2' },
+       { 'ead_id' => 'abc456', 'uri' => '/repositories/3/resources/4' }]
+    end
+
+    before do
```

Comment:
@corylown ignore this if I'm crossing my wires on the meaning of "repository" here -- but it seems like if you are querying for ` repository_id: 11`, it is confusing to have the example uris be in `/repositories/1/` and `/repositories/3/` -- ?

Reply:
Makes sense that this would be confusing. Fixed.